tests/framework_grpc/test_server.py

```diff
@@ -0,0 +1,183 @@
+import six
+import grpc
+import pytest
+from _test_common import create_request, wait_for_transaction_completion
+from newrelic.core.config import global_settings
+from testing_support.fixtures import (validate_transaction_metrics,
+        validate_transaction_event_attributes, override_application_settings,
+        override_generic_settings, function_not_called,
+        validate_transaction_errors)
+
+
+def select_python_version(py2, py3):
+    return six.PY3 and py3 or py2
+
+
+if hasattr(grpc, '__version__'):
+    GRPC_VERSION = tuple(int(v) for v in grpc.__version__.split('.'))
+else:
+    GRPC_VERSION = None
+
+is_lt_grpc118 = GRPC_VERSION is None or GRPC_VERSION < (1, 18)
+is_lt_grpc18 = GRPC_VERSION is None or GRPC_VERSION < (1, 8)
```

Comment:
We can probably remove this skip logic now that the oldest grpc version we're testing with is 1.25.